examples/query/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[workspace]
+members = [
+    "canisters/rust",
+]
+
+[profile.release]
+lto = true
+opt-level = 'z'
+codegen-units = 1
```

Comment:
We should have newlines at the end of our files, @dansteren can help you set up VS Code automation for this if you want. This comment goes for all files in this pull request